plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -76,10 +76,8 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
 
   @Suppress("UNCHECKED_CAST")
   private var gesturesPlugin: GesturesPlugin = delegateProvider.mapPluginProviderDelegate.getPlugin(
-    Class.forName(
-      PLUGIN_GESTURE_CLASS_NAME
-    ) as Class<GesturesPlugin>
-  )
+    MAPBOX_GESTURES_PLUGIN_ID
+  ) as GesturesPlugin?
```

Comment:
Considering we are using actual typed plugins, would it makes sense making getPlugin generic? This was we don't need to cast in these situations

Reply:
IMHO it's pretty the same in general but generic solution seems indeed a more clear one, I'll do that 👍